tests/components/version/test_sensor.py

```diff
@@ -47,7 +47,6 @@ async def async_setup_sensor_wrapper(
         await hass.async_block_till_done()
```

Comment:
There's a persistent_notification setup left on line 39.

Reply:
There were actually 2 (one in common.py as well) 👍